components/ui/Typography.jsx

```diff
@@ -1,35 +1,30 @@
-export const Heading1 = ({ children, className }) => {
-    return (
-        <h1 className={`md:text-23 md:leading-15.33 ${className}`}>{children}</h1>
-    )
-}
+import classNames from "classnames";
 
-export const Heading2 = ({ children, className }) => {
-    return (
-        <h2 className={`font-bold md:text-18.25 md:leading-22.75 leading-18.5 text-9.5 ${className}`}>{children}</h2>
-    )
-}
-
-export const Heading3 = ({ children, className }) => {
-    return (
-        <h3 className={`font-semibold md:text-14.75 text-9.5 md:leading-22.13 leading-14.25 ${className}`}>{children}</h3>
-    )
-}
-
-export const Heading4 = ({ children, className }) => {
-    return (
-        <h4 className={`md:text-11.75 md:leading-17.63 leading-14.25 text-9.5 text-white ${className}`}>{children}</h4>
-    )
-}
-
-export const Heading5 = ({ children, className }) => {
-    return (
-        <h5 className={`font-medium md:text-9.5 md:leading-12 leading-7.5 text-2xl ${className}`}>{children}</h5>
-    )
-}
+export const Heading = ({ children, className, type }) => {
+  return (
+    <div
+      className={classNames({
+        [`font-medium md:font-normal md:text-14.75 text-7.25 md:leading-22 leading-11 -tracking-0.02 ${className}`]:
+          type === "one",
+        [`font-normal md:text-11.75 md:leading-14.75 leading-7.15 text-6.5 text-white -tracking-0.015 md:-tracking-0.02 ${className}`]:
```

Comment:
A tip here: When you create such components it's better to always handle the default behavior. in this case, what would be the default type in case the type is not supplied? 

This can be done by giving a default value to the prop or using a switch case to handle different cases and convert one of the types to the default one.